runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc

```diff
@@ -198,8 +195,7 @@ void FullyConnectedLayer::prepare()
     }
   }
 
-#ifdef USE_RUY_GEMV
-#ifdef EXPERIMENTAL_RUY_FEATURE
+#if defined(__ARM_NEON__) && defined(USE_RUY_GEMV)
```

Comment:
if Ruy library is not used on x86, is `#if defined(__ARM_NEON__)` unnecessary?

Reply:
@YongseopKim Without `#if defined(__ARM_NEON__)`, constant tensor is deallocated even if it is not cached by ruy library on x86.

Reply:
Ah, It was meaning that - if`Ruy library is not used on x86`, the `USE_RUY_GEMV` would be not defined so that this code would not built. I guess your meanging is the ruy library is built and used on x86?

Reply:
FYI, the reason why I thought so is the word `USE_RUY_GEMV` expresses the availiability of ruy.

Reply:
@YongseopKim Ah, I understand your point. On x86 architecture, ruy library is built, and `USE_RUY_GEMV` is defined. Although ruy library is built, it is not used by fullyconneted layer on x86.

Reply:
Okay. If so, I think it's necessary to make ruy library not built on x86 regardless of this PR. (If the change would land, of course, this defined macro should be fixed.)